test/conftest.py

```diff
@@ -35,7 +35,7 @@
 def pytest_addoption(parser):
     parser.addoption('--docker-base-name', default='preprod-mxnet')
     parser.addoption('--region', default='us-west-2')
-    parser.addoption('--framework-version', default='1.2.1')
+    parser.addoption('--framework-version', default='1.3.0')
```

Comment:
The tests would fail with all the older version now. Should we mention it in README?

Reply:
not just the tests, but everything. I'll update the README when I remove the old Dockerfiles?

Reply:
that's fine with me.